pcp/columns/tty_name

```diff
@@ -0,0 +1,9 @@
+# pcp-htop(1) configuration file - see pcp-htop(5)
```

Comment:
Remove this file, not a useful DynamicColumn in practice - this is already a static column.